### PR TITLE
Handle deprecated locales

### PR DESCRIPTION
**Pull request by bsudekum**
Closes: https://github.com/mapbox/locale-utils/issues/4

This handles older, deprecated locales by inspecting the language component of a locale and seeing if it is one of the deprecated locales. If it is, the current locale is returned instead.

/cc @danpaz @1ec5

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -88,6 +90,14 @@ function parseLocaleIntoCodes (locale) {
     };
 }
 
+function handleDeprecatedLocales(language) {
```
Comment (1ec5) [inline index.js]: “Handle” isn’t a useful word to put in a function name. How about `getStandardLanguage()`?

File: lib/deprecated-locales.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = [{
+    previously: 'iw',
+    current: 'he'
```
Comment (1ec5) [inline lib/deprecated-locales.js]: This doesn’t need to be an array of objects. A simple object mapping deprecated codes to standard codes would do. If we need to attach more information to each entry in the future, we can use nested objects again. But for now it just makes `getStandardLanguage()` more complicated and less performant.

File: test.js [hunk 3]
```diff
@@ -22,6 +25,11 @@ tape('test bestMatchingLocale', function(t) {
     t.equal(locale.bestMatchingLocale('es-mx', availableLocales), 'es');
     t.equal(locale.bestMatchingLocale('zh-Hans-region', availableLocales), 'zh-Hans');
     t.equal(locale.bestMatchingLocale('pt-BR', availableLocales), 'pt-PT');
+
+    t.equal(locale.bestMatchingLocale('iw', availableLocales), 'he', 'Test deprecated locale');
```
Comment (1ec5) [inline test.js:29]: Add a test case for a deprecated language plus a country code, such as `iw-IL`.

**Comment by bsudekum** [comment]
@1ec5 @danpaz updated tests.

**Review by danpaz — APPROVED** [review]
LGTM :+1:

**Pull request merged**